modules/cells/src/main/java/dmg/cells/nucleus/CellAdapter.java

```diff
@@ -758,15 +758,15 @@ public void prepareRemoval(KillEvent killEvent)
     @Override
     public void postRemoval(KillEvent ce)
     {
-        _log.info("CellAdapter : prepareRemoval : waiting for gate to open");
+        logger.info("CellAdapter : prepareRemoval : waiting for gate to open");
```

Comment:
"Jeder Jeck ist anders". The main goal of the style is what we use as a team. Currently, we prefer LOGGER vs logger. I am ok with other places, where you change **_log** or **log** to logger.